pytorch_forecasting/models/dlinear/_dlinear_v2.py

```diff
@@ -0,0 +1,305 @@
+"""
+LTSF-DLinear model for Pytorch Forecasting.
+-------------------------------------------
+"""
+
+#################################################
+# NOTE: This is an experimental implementation  #
+# of LTSF-DLinear model for PTF v2.             #
+# It is an unstable API and subject to change.  #
+#################################################
+
+from typing import Any, Optional, Union
+import warnings
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from torch.optim import Optimizer
+
+from pytorch_forecasting.layers.decomposition import SeriesDecomposition
+from pytorch_forecasting.metrics import QuantileLoss
+from pytorch_forecasting.models.base._tslib_base_model_v2 import TslibBaseModel
+
+
+class DLinearModel(TslibBaseModel):
```

Comment:
rename this simply `DLinear`? `from models import DLinear` already implies that it is a model